tests/foreman/ui/test_openscap.py

```diff
@@ -101,9 +240,15 @@ def test_search_content(self):
 
         @Feature: OpenScap - Search.
 
+        @steps:
+
+        1. Create an openscap content.
+        2. Provide valid data-stream.xml file available from
+        scap-security-guide. (ssg-rhel6-ds.xml)
```

Comment:
Nitpick: indent this.

    1. Create an openscap content.
    2. Provide valid data-stream.xml file available from
       scap-security-guide. (ssg-rhel6-ds.xml)